regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
+                                "required": False,
+                                "type": "string"},
+                "scope": {"description": "",
+                          "required": False,
+                          "type": "list"},
+                "platform": {"description": "description of how and where the audience will access "
+                                "the deliverable.  e.g. Journal if it is a paper",
+                             "required": False,
+                             "type": "string"},
+                "rollout": {"description": "steps that the audience will take to access "
+                                           "and interact with the deliverable, "
+                                           "not needed for paper submissions",
+                            "required": False,
+                            "type": "list"},
+                "notes": {"description": "deliverable note",
+                          "required": False,
+                          "type": "list"},
+                "status": {"description": "current state of deliverable",
+                           "required": False,
+                           "type": "string",
+                           "eallowed": PROJECTA_STATUS},
+            }
+        },
+        "description": {
+            "description": "explanation of projecta",
+            "required": False,
+            "type": "string"
+        },
+        "end_date": {
+            "description": "projecta end date, yyyy-mm-dd. "
+                           "Not necessarily the same as deliverable due date",
+            "required": False,
+            "anyof_type": ["date", "string"]
+        },
+        "grants":{
+            "description": "grant id funding the project",
+            "required": False,
+            "type": "string"
+        },
+        "group_members": {
+            "description": "group member id's working on this project",
+            "required": False,
+            "type": "list"
+        },
+        "kickoff":{
+            "description": "details the projecta kickoff meeting",
+            "required": False,
+            "type": "dict",
+            "schema": {
+                "date": {"description": "kickoff meeting date, yyyy-mm-dd.",
+                         "required": False,
+                         "anyof_type": ["date", "string"]},
+                "due_date": {"description": "kickoff meeting by date, yyyy-mm-dd.",
+                             "required": False,
+                             "anyof_type": ["date", "string"]},
+                "name": {"description": "name of meeting",
+                         "required": False,
+                         "type": "string"},
+                "objective": {"description": "goal of the meeting",
+                              "required": False,
+                              "type": "string"},
+                "audience": {"description": "rolls of audience members "
```

Comment:
"list of people attending the meeting.  Normally this list is group_members, collaborators, and pi, or some subset of these.  if people are invited who are not already in these groups their names or id's can be added explicitly to the list"